redhat-appstudio/service-provider-integration-operator
# Script that helps to setup monitoring stack

### What does this PR do?
The script that helps to setup monitoring stack

### Screenshot/screencast of this PR
n/a

### What issues does this PR fix or reference?
metrics development environment for https://issues.redhat.com/browse/SVPI-102


### How to test this PR?
- deploy SPI on minikube
- deploy monitoring stack
- open grafana. 3 dashboards should be available: Prometheus, controller runtime, go metrics.
